# center of measurements
float64[] center
# covariance of last 10 measurements
float64[] matrix